conf/modules.config

```diff
@@ -465,6 +465,34 @@ process {
         ]
     }
 
+    withName: ANGSD_DOCOUNTS {
+        ext.args = [
+            "-iCounts 1",
+            "-r ${params.contamination_estimation_angsd_chrom_name}:${params.contamination_estimation_angsd_range_from}-${params.contamination_estimation_angsd_range_to}",
+            "-minMapQ ${params.contamination_estimation_angsd_mapq}",
+            "-minQ ${params.contamination_estimation_angsd_minq}"
+        ].join(' ').trim()
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: ANGSD_CONTAMINATION {
+        publishDir = [
+            path: { "${params.outdir}/contamination_estimation/angsd/"},
+            mode: params.publish_dir_mode,
+            pattern: '*.txt'
+        ]
+    }
+
+    withName: PRINT_CONTAMINATION_ANGSD {
+        publishDir = [
+            path: { "${params.outdir}/contamination_estimation/angsd/"},
+            mode: params.publish_dir_mode,
+            pattern: 'nuclear_contamination*'
```

Comment:
I think the `pattern` can even be omitted? JSON and TXT are all the outputs, no?

Reply:
If I remove the pattern, it also publishes the versions.yml